src/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if (APPLE)
   set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} -multiply_defined suppress -Wl,-undefined,dynamic_lookup -Wl,-dead_strip_dylibs -bundle_loader ${PG_BINDIR}/postgres")
 endif (APPLE)
 
-include_directories (${CMAKE_CURRENT_BINARY_DIR} ${PG_INCLUDEDIR} ${PG_INCLUDEDIR_SERVER})
+include_directories (${CMAKE_CURRENT_BINARY_DIR} ${PG_INCLUDEDIR_SERVER})
```

Comment:
${PG_INCLUDEDIR} is actually the libpq include files which aren't necessary for our builds and are often out-of-sync with the files for the server development files on Ubuntu and debian. This Is why I removed it.

Reply:
It's needed for building on Windows, so you'll need to special case WIN32 instead of getting rid of it.

Reply:
@RobAtticus what errors do you see on windows otherwise?

Reply:
Added it below. Can you try it now?